src/getCoinCombination.test.js

```diff
@@ -7,4 +7,52 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return an array`, () => {
+    expect(typeof getCoinCombination(18)).toBe('object');
+  });
+
```

Comment:
only my opinion: for the array check, we have a separate special check:
`     expect(Array.isArray(getCoinCombination())).toBe(true);`
but your check is also correct, so everything is fine

Reply:
> 

nice! thanks for advice! i'll keep it in mind next time